modular_skyrat/modules/food_replicator/replicator_designs/replicator_food.dm

```diff
@@ -0,0 +1,111 @@
+/datum/design/ration
+	name = "foreign colonization ration"
+	id = "slavic_mre"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 550)
+	build_path = /obj/item/storage/box/colonial_rations
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/pljeskavica
+	name = "foreign colonization ration, main course"
+	id = "slavic_burger"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 200)
+	build_path = /obj/item/food/colonial_course/pljeskavica
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/nachos
+	name = "foreign colonization ration, side dish"
+	id = "mexican_chips"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 150)
+	build_path = /obj/item/food/colonial_course/nachos
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/blins
+	name = "foreign colonization ration, dessert"
+	id = "slavic_crepes"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 100)
+	build_path = /obj/item/food/colonial_course/blins
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/spork
+	name = "foreign colonization ration, utensils"
+	id = "slavic_utens"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 75)
+	build_path = /obj/item/storage/box/utensil
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_FOOD)
+
+/datum/design/fasani
```

Comment:
fasani? That's not really a name that's going to help anyone figure out what the typepath of this entry is actually supposed to mean.

Reply:
its joever